google/cloud/bigquery/model.py

```diff
@@ -433,6 +447,66 @@ def __repr__(self):
             self.project, self.dataset_id, self.model_id
         )
 
+class TransformColumn:
+    """TransformColumn represents a transform column feature.
+
+    See
+    https://cloud.google.com/bigquery/docs/reference/rest/v2/models#transformcolumn
+
+    Args:
+        column_name:
```

Comment:
Update these args to match the args of the `__init__` function. (Note: you can omit "self" from the docstring, as that's implied.)